### ISSUE TITLE
Error in py.test since pyethereum update 1.6.1 -> 2.0.4 

### ISSUE THREAD

**Issue by ice09** [ethereum/populus#300]
* Populus Version: 1.6.9 and 2.0.0a4
* OS: osx

Steps to reproduce:
1. `pip install populus==2.0.0a4` (or `pip install populus`)
2. `populus init`
3. `populus compile`
4. `py.test`

results in 

```
    import sys
    import logging
    
    import rlp
    
    try:
        from sha3 import keccak_256
    except ImportError:
        from sha3 import sha3_256 as keccak_256
    
>   from ethereum import blocks
E   ImportError: cannot import name 'blocks'

../../venvs/tdd/lib/python3.6/site-packages/testrpc/rpc.py:11: ImportError
```

when using `pip uninstall ethereum` followed by `pip install ethereum==1.6.1` everything is fine again.
The error is caused by the pyethereum update, but is shows in the populus tests.

**Comment by rstormsf** [comment]
I'm getting the same issue

**Comment by pipermerriam** [comment]
Work to support the newer release of pyethereum will be underway very soon.  For now you should stick to `1.6.x`.

**Comment by pipermerriam** [comment]
This should be fixed in fresh installs as of #303 but it's not yet included in a release.  Stay tuned.

**Comment by veox** [comment]
OK to close?..
